protected/humhub/modules/user/models/Invite.php

```diff
@@ -112,6 +112,11 @@ public function beforeSave($insert)
 
     public function selfInvite()
     {
+        if (Yii::$app->settings->get('maintenanceMode')) {
+            Yii::$app->getView()->warn(Yii::t('error', 'Only admins have an access to the site on maintenance mode.'));
```

Comment:
Here as well `an` can be removed

Reply:
Commit https://github.com/humhub/humhub/pull/4773/commits/23a9ed3c0a5750e569d9480669fc7fe0d6f52ae1